### PR TITLE
Farsi implementation

### PR DESCRIPTION
**Pull request by HEskandari**

### CODE REVIEW

**Comment by ploeh** [comment]
Thank you for your interest in contributing to Numsense. Before I proceed with a review, could you please update the fork so that all merge conflicts are resolved?

Once you've done that, please add a note here to that effect. The reason is that GitHub doesn't send out notifications when pull request forks are updated. Adding a note here ensures that I get a notification, and that means that I'll discover the update sooner.

**Comment by HEskandari** [comment]
Fixed the conflicts. 

Thanks beforehand.

File: Numsense.UnitTests.CSharp/NumeralTests.cs [hunk 1]
```diff
@@ -34,5 +34,13 @@ public void DanishIsSingleton()
             var actual = Numeral.Danish;
             Assert.Same(expected, actual);
         }
+
```
Comment (ploeh) [inline Numsense.UnitTests.CSharp/NumeralTests.cs]: Is there an explicit reason you left out a `FarsiIsCorrect` test?

**Comment by ploeh** [comment]
Thank you for your interest in contributing to Numsense! This pull request certainly looks promising :+1: 

I only have a few small questions (apart from the inline comment I've already added):

- Is there a specific reason you didn't add properties to the `NumeralProperties` module (`NumeralProperties.fs`)? For Danish and English, this is how all negative numbers are verified, but Farsi negative numbers may be dealt with using logic different from that, so there may be a good reason for that; I only ask out of ignorance. Likewise, the other property of both English and Danish is that numbers can round-trip, but again, Farsi numerals may be structured in such a way that this is logically impossible...
- Did you forget to add `FarsiNumeralConverter` to `NumeralConverterProperties`, or is there a specific reason you left that out?

File: Numsense.UnitTests.CSharp/NumeralTests.cs (hunk 1 above)
Comment (HEskandari) [inline Numsense.UnitTests.CSharp/NumeralTests.cs]: No, just my ignorance :) I'll update the PR with the missing tests.

**Comment by HEskandari** [comment]
The latest commit should have addressed all the issues mentioned above. Please let me know if there's anything else.

**Comment by ploeh** [comment]
It's looking good now :+1: It also builds and runs all tests without warnings on my machine, so from a technical perspective I think we're good to go.

As I don't read Farsi, I'd like someone who does to review that part of it, if at all possible. I don't expect any errors, but it's always good with a second pair of eyes :smile:

**Comment by ploeh** [comment]
I solicited a review on Twitter: https://twitter.com/ploeh/status/685066949114609664

Please retweet and spread the word :smile:

**Comment by ploeh** [comment]
I suppose no complaints means that there are no errors found so far :smile: 

The only problem now is that the PR no longer can be automatically merged. If you can update the PR by addressing the conflicts, I'll subsequently pull it.

**Comment by HEskandari** [comment]
Just merged the branch with master. Please pull.

**Comment by ploeh** [comment]
Thank you for your contribution! It's now live as [Numsense 0.6.0](https://www.nuget.org/packages/Numsense/0.6.0).

**Pull request merged**